src/frontend/sharedComponents/HomeHeader.tsx

```diff
@@ -1,61 +1,102 @@
 import React from 'react';
 import {View, StyleSheet} from 'react-native';
-import LinearGradient from 'react-native-linear-gradient';
-import {IconButton} from './IconButton';
-import {GPSPill} from './GPSPill';
 import {BottomTabHeaderProps} from '@react-navigation/bottom-tabs';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
-import {DrawerMenuIcon} from './icons/DrawerMenuIcon';
-import SyncIconCircle from '../images/Sync.svg';
+import {defineMessages, useIntl} from 'react-intl';
+
+import DeviceIcon from '../images/DeviceIcon.svg';
+import {IconButton} from './IconButton';
+import {HeaderText} from './Text/HeaderText';
+import {BLUE_GREY, WHITE} from '../lib/styles';
+import {useProjectSettings} from '../hooks/server/projects';
+
+const m = defineMessages({
+  mySoloProject: {
+    id: 'homeHeader.header.mySoloProject',
+    defaultMessage: 'My Solo Project',
+  },
+});
+
+type HomeHeaderProps = BottomTabHeaderProps & {
+  openDrawer: () => void;
+  backgroundColor?: string;
+  showBottomBorder?: boolean;
+};
 
-export const HomeHeader = ({
-  navigation,
+export function HomeHeader({
   openDrawer,
-}: BottomTabHeaderProps & {openDrawer: () => void}) => {
+  backgroundColor = 'transparent',
+  showBottomBorder = false,
+}: HomeHeaderProps) {
   const insets = useSafeAreaInsets();
+  const {formatMessage} = useIntl();
+  const {data} = useProjectSettings();
+
+  const projectName = data?.name ? data.name : formatMessage(m.mySoloProject);
 
   return (
-    <View style={[styles.header, {paddingTop: insets.top}]}>
-      <LinearGradient
-        style={styles.linearGradient}
-        colors={['#0006', '#0000']}
-      />
-      <IconButton
-        style={styles.leftButton}
-        accessibilityLabel="Go to Sync Screen"
-        onPress={() => {
-          navigation.navigate('Sync');
-        }}>
-        <SyncIconCircle testID="MAIN.sync-icon" />
-      </IconButton>
-      <GPSPill navigation={navigation} />
-      <DrawerMenuIcon
-        style={{marginRight: 20}}
-        onPress={openDrawer}
-        testID="drawer-icon-home"
-        accessibilityLabel="Open Navigation Drawer"
-      />
+    <View
+      style={[
+        styles.container,
+        {
+          paddingTop: insets.top,
```

Comment:
The padding looks a little uneven, looks like there is more space below. perhaps `justifyContent:'center'` (that is just a guess)

Reply:
<img width="292" alt="image" src="https://github.com/user-attachments/assets/ef0e2093-e29a-49d0-9121-2267a6d6ea67" />